lib/kernel/host/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@
 set(SOURCES_WITH_SLEEF abs.cl
 abs_diff.cl
 add_sat.cl
+addrspace_operators.ll
```

Comment:
I only tested the non-SLEEF version. I also wasn't sure how / where to add this exactly.
This might also have to be part of the generate.rb script. I'd appreciate if someone else could do the SLEEF changes.

Reply:
Also note that https://github.com/pocl/pocl/commit/4500774f4af2a134e027169b199d3dcd7195cf20 explicitly removed more specialized variants of these functions from the host backend.